README.md

```diff
@@ -42,8 +42,9 @@ Run `npm install` in `CodeLabz` folder.
  cd CodeLabz/
  npm install
 ```
+> **NOTE**: If `npm install` is not executing successfully, then use node v14 and try again.
```

Comment:
rephrase it as:

run `npm install --legacy-peer-deps` with node v14

Reply:
done